### PR TITLE
Update 1-prepare-artwork.md

### PR DESCRIPTION
**Pull request by DanLShane**
Part one checked for grammar, spelling, and punctuation.  I altered the **Notes!** and **Warning!** standouts to match American English standards, but you can change them back if they match your style better.

### CODE REVIEW

File: VisualPinball.Unity/Documentation~/creators-guide/tutorials/relatistic-plastics/1-prepare-artwork.md [hunk 1]
```diff
@@ -8,56 +8,56 @@ description: How to setup the plastics artwork so we can easily regenerate the t
...
 
-As mentioned in the introduction, we're aiming for a single texture that contains all plastics. The advantage is that this way, less texture data needs to be transferred to the graphics card and only one material instance must be instantiated. It also allows you to process everything together, versus having to repeat this workflow for every plasic separately.
+As mentioned in the introduction, we're aiming for a single texture that contains all plastics. The advantage is that this way, less texture data needs to be transferred to the graphics card and only one material instance must be instantiated. It also allows you to process everything together, versus having to repeat this workflow for every plastic separately.
 
 ## Step 1: Create Artwork
 
 There are many ways to create artwork. If you're working on an original game, you might draw it using a drawing software. For recreations where you have access to the physical machine, you might disassemble the plastics and scan them or take pictures with a good camera. If you don't have access to the physical plastics, you might get good sources by searching on the internet.
 
-> [!note]
+> [Warning!]
```
Comment (freezy) [inline VisualPinball.Unity/Documentation~/creators-guide/tutorials/relatistic-plastics/1-prepare-artwork.md:17]:
```suggestion
> [!note]
```

File: VisualPinball.Unity/Documentation~/creators-guide/tutorials/relatistic-plastics/1-prepare-artwork.md [hunk 2]
```diff
@@ -8,56 +8,56 @@ description: How to setup the plastics artwork so we can easily regenerate the t
...
 
-Once you have an image of the plastic, you need to decide whether you will be using it as-is, or whether you're going to redraw it. Depending on the type of artwork, redrawing might not possible or a huge effort, and "cleaning it up" in Photoshop will must do it. The advantage of redrawing it is that you can produce arbitrary large textures, and make the colors match perfectly.
+Once you have an image of the plastic, you need to decide whether you will be using it as-is, or whether you're going to redraw it. Depending on the type of artwork, redrawing might not possible or a huge effort, and "cleaning it up" in Photoshop will do it. The advantage of redrawing it is that you can produce arbitrary large textures and make the colors match perfectly.
 
-> [!note]
-> Speaking of colors, it's best to use PBR-compatibe colors, meaning your texture shouldn't contain absolute blacks or whites, and the brightness of other colors should be at around 95% max. For black, we're using `#272727` and for white `#f4f5f0`.
+> [Note!]
```
Comment (freezy) [inline VisualPinball.Unity/Documentation~/creators-guide/tutorials/relatistic-plastics/1-prepare-artwork.md:27]:
```suggestion
> [!note]
```

File: VisualPinball.Unity/Documentation~/creators-guide/tutorials/relatistic-plastics/1-prepare-artwork.md [hunk 3]
```diff
@@ -8,56 +8,56 @@ description: How to setup the plastics artwork so we can easily regenerate the t
...
 
 ![Artwork in Adobe Illustrator](illustrator-separate-artwork.png)
 
 Note the pink outlines. They sit on a separate layer and can be easily toggled. They should contain drill holes for the screws as well, as they are visible due to the plastic being transparent and the ink actually sitting below the plastic.
 
-Since our first step is to create the texture, hide the outlines and save all of your files.
+Since our first step is to create the texture, hide the outlines and save all your files.
 
-> [!note]
+> [Note!]
```
Comment (freezy) [inline VisualPinball.Unity/Documentation~/creators-guide/tutorials/relatistic-plastics/1-prepare-artwork.md:38]:
```suggestion
> [!note]
```

File: VisualPinball.Unity/Documentation~/creators-guide/tutorials/relatistic-plastics/1-prepare-artwork.md [hunk 4]
```diff
@@ -8,56 +8,56 @@ description: How to setup the plastics artwork so we can easily regenerate the t
...
 
-Since we're creating the texture, we don't need the pink outlines. So before continueing, open all of your source files, hide them, and save them.
+Since we're creating the texture, we don't need the pink outlines. So, before continuing, open all of your source files, hide them, and save them.
 
-As we might update our original files later, we won't copy and paste our plastics into this document, but *link them*. In order to do this, go to *File -> Place...* and select all your plastics. Then arrange them in a way so there's as little white space as possible remaining. 
+As we might update our original files later, we won't copy and paste our plastics into this document but *link them* instead. In order to do this, go to *File -> Place...* and select all your plastics. Then arrange them in a way so there's as little white space as possible remaining. 
 
-> [!note]
-> In order for the plastics to have a similar pixel density, we recommend keeping the proportional size of the plastics approximately the same, i.e. larger plastics should take more space than smaller ones.
+> [Note!]
```
Comment (freezy) [inline VisualPinball.Unity/Documentation~/creators-guide/tutorials/relatistic-plastics/1-prepare-artwork.md:51]:
```suggestion
> [!note]
```

File: VisualPinball.Unity/Documentation~/creators-guide/tutorials/relatistic-plastics/1-prepare-artwork.md [hunk 5]
```diff
@@ -8,56 +8,56 @@ description: How to setup the plastics artwork so we can easily regenerate the t
...
 
 Your artboard should now look something like this:
 
 ![Joined artwork in Adobe Illustrator](illustrator-joined-artwork.png)
 
-Note the *Links* toolbox that shows the individual plastic files. Also note the pink surfaces on the right, outside the artboard. These are plastics that don't have any artwork, i.e. blank, transparent pieces. We keep them here because we want to generate meshes as well, but don't need it in the texture, which is defined by the artboard.
+Note the *Links* toolbox that shows the individual plastic files. Also note the pink surfaces on the right, outside the artboard. These are plastics that don't have any artwork, i.e., blank, transparent pieces. We keep them here because we want to generate meshes as well, but don't need it in the texture, which is defined by the artboard.
 
-> [!warning]
+> [Warning!]
```
Comment (freezy) [inline VisualPinball.Unity/Documentation~/creators-guide/tutorials/relatistic-plastics/1-prepare-artwork.md:60]:
```suggestion
> [!warning]
```

File: VisualPinball.Unity/Documentation~/creators-guide/tutorials/relatistic-plastics/1-prepare-artwork.md [hunk 6]
```diff
@@ -69,16 +69,16 @@ We'll first export the texture. In Illustrator, click on *File -> Export -> Expo
 
 Since you've set the resolution to 4096 × 4096 pixels when you've created the document, and the default pixel density is 72dpi, choose 72dpi. This should result in a PNG file at 4096 × 4096 pixels.
 
-> [!note]
-> When I exported it when writing this tutorial, Illustrator exported it as 4097 × 4097 🤦‍♂. So I ended up exporting it with a slightly higher resolution and sizing it down in Photoshop.
+> [Note!]
```
Comment (freezy) [inline VisualPinball.Unity/Documentation~/creators-guide/tutorials/relatistic-plastics/1-prepare-artwork.md:72]:
```suggestion
> [!note]
```

File: VisualPinball.Unity/Documentation~/creators-guide/tutorials/relatistic-plastics/1-prepare-artwork.md [hunk 7]
```diff
@@ -69,16 +69,16 @@ We'll first export the texture. In Illustrator, click on *File -> Export -> Expo
...
 ### Outlines
 
-The goal of having outlines in the artwork is so that we can extract the mesh from it, and allows us to align the artwork perfectly on the surface. We're going to export the outlines as solid surfaces in the SVG format. The result we can then import into Blender.
+The goal of having outlines in the artwork is so that we can extract the mesh from it, and it allows us to align the artwork perfectly on the surface. We're going to export the outlines as solid surfaces in the SVG format. The result we can then import into Blender.
 
-In order to do that, open all your artwork files, hide the actual artwork and only show the outline. Make it **one single solid surface per plastic**, meaning if you have holes, you need to remove them from the parent curve.
+In order to do that, open all your artwork files, hide the actual artwork, and only show the outline. Make it **one single solid surface per plastic**, meaning if you have holes, you need to remove them from the parent curve.
 
-> [!note]
+> [Note!]
```
Comment (freezy) [inline VisualPinball.Unity/Documentation~/creators-guide/tutorials/relatistic-plastics/1-prepare-artwork.md:81]:
```suggestion
> [!note]
```

**Review by freezy — CHANGES_REQUESTED** [review]
Thanks a lot! The `[!note]` and `[!warning]` tags are keywords and are interpreted by the documentation site builder.

I've added suggestions to change them back, I think you can just batch them and commit.

**Comment by DanLShane** [comment]
All committed.

 

--

 

Be seeing you...

**Review by freezy — APPROVED** [review]
Thanks a lot!

**Pull request merged**